pkg/storaged/dialog.jsx

```diff
@@ -1094,7 +1097,7 @@
                         ? <p>
                             <b>{_("Processes using the location")}</b>
                             <List>
-                                { processes.slice(0, max).map((u, i) => <ListItem key={i}>{u.comm} (user: {u.user}, pid: {u.pid})</ListItem>) }
+                                { processes.slice(0, max).map((u, i) => <ListItem key={i}>{u.comm} (user: {u.user}, pid: {mock?.pid ?? u.pid})</ListItem>) }
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17654-20230322-115306-eab6c63e-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)